mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/print/config.yaml

```diff
@@ -37,7 +37,7 @@ scales:
   - 559082264
 
 # see https://github.com/georchestra/georchestra/issues/362
-brokenUrlPlaceholder: http://sdi.georchestra.org/public/transparent.gif
+brokenUrlPlaceholder: https://sdi.georchestra.org/public/transparent.gif

```

Comment:
Just as a reminder:

- This can cause problems with old JVM (the one used in pigma is too old for certain HTTPS website - https://github.com/camptocamp/georchestra-pigma-configuration/issues/215 (private))

- More relevant point (since the doc mentionned letsencrypt above): The letsencrypt certificates might not be trusted by the JVM: http://stackoverflow.com/questions/34110426/does-java-support-lets-encrypt-certificates